packages/react-next/src/components/Slider/useSlider.ts

```diff
@@ -0,0 +1,264 @@
+import * as React from 'react';
+import { ISliderProps, ISliderStyleProps, ISliderStyles, ISliderState } from './Slider.types';
+import { useId, useBoolean, useControllableValue } from '@uifabric/react-hooks';
+import { KeyCodes, css, getRTL, getRTLSafeKeyCode, on } from '../../Utilities';
+import { classNamesFunction, getNativeProps, divProperties } from '../../Utilities';
+
+export const ONKEYDOWN_TIMEOUT_DURATION = 1000;
+
+const getClassNames = classNamesFunction<ISliderStyleProps, ISliderStyles>({
+  useStaticStyles: true,
+});
+
+export const useSlider: (props: ISliderProps, ref: React.Ref<HTMLDivElement>) => ISliderState = (props, ref) => {
+  const {
+    step = 1,
+    className,
+    disabled = false,
+    label,
+    max = 10,
+    min = 0,
+    showValue = true,
+    buttonProps = {},
+    vertical = false,
+    valueFormat,
+    styles,
+    theme,
+  } = props;
+
+  const ariaLabel = props['aria-label'];
```

Comment:
You can destructure this above by doing `'aria-label': ariaLabel.

Reply:
Neat! I didn't know that.